core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -537,6 +538,26 @@ public void removeBlock(long sessionId, long blockId)
     mLocalBlockStore.removeBlock(sessionId, blockId);
   }
 
+  @Override
+  public void removeAllBlock(long sessionId)
+      throws InvalidWorkerStateException, IOException {
+    BlockStoreMeta mBlockStoreMeta = getStoreMetaFull();
+    for (List<Long> mBlockIds : mBlockStoreMeta.getBlockList().values()) {
```

Comment:
As you want to remove all block, so you can remove all block files in the cache storage tier location and clear all the blocks in `mLocalBlockStore`, any thoughts?